doc/whatsnew/2.14.rst

```diff
@@ -97,6 +97,30 @@ Other Changes
 
   Ref #5392
 
+* ``UnsupportedAction`` has been deprecated.
+
+  Ref #5392
+
+* ``OptionsManagerMixIn`` has been deprecated.
```

Comment:
This is highly satisfactory. I'm not going to comment on each mixin but I really hated those classes 😄

Reply:
Me too! `mypy` despises them too!